crates/ra_lsp_server/src/main_loop/handlers.rs

```diff
@@ -749,6 +750,29 @@ pub fn handle_code_lens(
             }),
     );
 
+    lenses.extend(
```

Comment:
This will be moved eventually into a separate  LSP requestion right?

Reply:
Right, removed now to avoid confusion.
I use it currently to do some sort of a final integration testing.